packages/lab/src/components/Flow/stories/Flow.stories.tsx

```diff
@@ -160,3 +162,18 @@ export const DynamicHandles: StoryObj<HvFlowProps> = {
   },
   render: () => <DynamicHandlesStory />,
 };
+
+export const InvalidFlow: StoryObj<HvFlowProps> = {
+  parameters: {
+    docs: {
+      description: {
```

Comment:
We could add a description to contextualise the sample

Reply:
Of course, I'll add.